storybook/src/admin/form/AllFieldComponents.tsx

```diff
@@ -65,6 +66,7 @@ function Story() {
                                         fullWidth
                                     />
                                     <SwitchField name="switch" label={values.switch ? "On" : "Off"} fieldLabel="Switch" />
+                                    <CurrencyField name="currency" label="Currency" />
```

Comment:
Maybe move this above the `CheckboxField` and add the `fullWidth` prop. 
The field looks a bit lost, next to the switch.